### ISSUE TITLE
Kadane's Algorithm Explanation and Implementation in C++

### ISSUE THREAD

**Issue by vanshikapandey** [HarshCasper/NeoAlgo#4042]
Kadane's Algorithm is one of the widely asked algorithm in interviews. I have a good knowledge of C++
 and fair understanding of the algorithm. I would like to work under this issue and implement it. Please **/assign** it to me underGSSOC'21 @ project-lead

**Comment by vanshikapandey** [comment]
/assign

Thanks for assigning sir

**Comment by ankitaggarwal23** [comment]
@vanshikapandey Already someone did this problem in C++ section. Kindly don't open such issue always check before opening a new issue.
